### PR TITLE
rename all res to self.res

### PR DESCRIPTION
**Pull request by myakove**
Rename all res to self.res to be consistent with the code.

### CODE REVIEW

File: ocp_resources/restore.py [hunk 1]
```diff
@@ -39,16 +39,16 @@ def __init__(
         self.backup_name = backup_name
 
     def to_dict(self):
-        res = super().to_dict()
+        self.res = super().to_dict()
```
Comment (rnetser) [inline ocp_resources/restore.py:42]: `self.res` be defined in __init__  (i.e not hide attributes)

File: ocp_resources/backup.py [hunk 2]
```diff
@@ -50,5 +50,5 @@ def to_dict(self):
         )
 
         if self.excluded_resources:
-            res["spec"]["excludedResources"] = self.excluded_resources
-        return res
+            self.res["spec"]["excludedResources"] = self.excluded_resources
+        return self.res
```
Comment (rnetser) [inline ocp_resources/backup.py:54]: now that is `self.res` is updated, do we still need to return it?

**Comment by myakove** [comment]
!DO-NOT-MERGE

**Comment by myakove** [comment]
!-DO-NOT-MERGE

File: ocp_resources/virtual_machine_import.py [hunk 3]
```diff
@@ -306,4 +308,4 @@ def to_dict(self):
                     _map_mappings(mappings=mapping.storage_mappings),
                 )
 
-        return res
+        return self.res
```
Comment (rnetser) [inline ocp_resources/virtual_machine_import.py:311]: can be removed

File: ocp_resources/virtual_machine_import.py [hunk 4]
```diff
@@ -289,12 +289,14 @@ def __init__(
         self.mapping = mapping
 
     def to_dict(self):
-        res = super().to_dict()
+        super().to_dict()
         if self.yaml_file:
-            return res
+            return self.res
```
Comment (rnetser) [inline ocp_resources/virtual_machine_import.py:294]: `if not self.yaml_file`

**Review by rnetser — CHANGES_REQUESTED** [review]

**Comment by myakove** [comment]
!verified

**Review by rnetser — APPROVED** [review]

**Comment by myakove** [comment]
!verified

**Comment by myakove** [comment]
!cherry-pick v4.12

**Comment by myakove** [comment]
!cherry-pick v4.12

**Pull request merged**